packages/typescript/src/symbols/ts-module-scope.ts

```diff
@@ -19,7 +19,11 @@ export interface TSModuleScope extends OutputScope {
    */
   importedSymbols: Map<TSOutputSymbol, TSOutputSymbol>;
   importedModules: ImportRecords;
-  addImport(symbol: TSOutputSymbol, module: TSModuleScope): TSOutputSymbol;
+  addImport(
+    symbol: TSOutputSymbol,
+    module: TSModuleScope,
+    type?: boolean,
```

Comment:
Should maybe have an option bag instead of boolean positional.

It didn't feel right to have this be the symbol flag as they don't really apply here.

Reply:
Yeah I think options bag here too (though I care less since this is pretty much internal)